src/platform/baytrail/memory.c

```diff
@@ -109,3 +110,8 @@ struct mm memmap = {
 	.total = {.free = HEAP_SYSTEM_SIZE + HEAP_SYS_RUNTIME_SIZE +
 			HEAP_RUNTIME_SIZE + HEAP_BUFFER_SIZE,},
 };
+
+void platform_init_memmap(void)
+{
+	/*  memmap has been initialized already */
```

Comment:
pls consider replacing "already" with "statically as part of .data".

Reply:
done